### DISCUSSION TITLE
Collection Tags being removed -- Is Plex-Meta-Manager doing this?

### DISCUSSION THREAD

**Discussion by j-ktz** [meisnate12/Plex-Meta-Manager#198] [Q&A] [answered]
So when I download movies, I'll manually add a Collection tag. What's strange, is later when I check the details, the Collection tag is gone, but the orange padlock is still set to lock from when I added the tag. How is it being removed? It happens on multiple movies, so not isolated to one incident. I'm not manually refreshing data or anything like that. Any ideas? I see that PMM is still adding Collection tags to some movies, but wondering if my manually added ones being removed is due to PMM somehow? Or maybe it's a Plex issue? Just strange... ideas?

**Comment by meisnate12** [comment, marked as answer]
PMM can remove collection tags from movie if the collection has sync_mode set to sync. can you send me your meta.log

**Reply by j-ktz** [reply]
My sync_mode is set to append, so not sure. Here are my logs! 
[meta-25.log](https://github.com/meisnate12/Plex-Meta-Manager/files/6291597/meta-25.log)

**Reply by meisnate12** [reply]
youve got a few errors in your log your alot of BadRequest Errors which is really weird and you also Reddit Top Movies Collection is failing because of an invalid collection order

can you update PMM to the latest version and update PMS to the latest version

**Reply by j-ktz** [reply]
PMS is on the latest version. I'll update PMM. What should I set sync_mode to for movie collections, append?

**Reply by meisnate12** [reply]
for me it depends on the collection but I usually use sync_mode: sync

**Reply by j-ktz** [reply]
Okay. My config says append, but my Movies.yml says sync. I just don't want the collection tags I manually add to be removed. For example, I'll manually tag a movie LBGT+, and then when the script runs, the tag is removed, but never added back by any other lists.

**Reply by meisnate12** [reply]
your LGBT+ collection must have sync_mode: append
